tests/attendee/attendee-repository.php

```diff
@@ -239,9 +253,9 @@ public function test_is_new_translation_contributor() {
 		$this->repository->insert_attendee( $attendee12 );
 		$this->repository->insert_attendee( $attendee13 );
 
-		$this->assertTrue( $attendee11->is_new_contributor() );
-		$this->assertTrue( $attendee12->is_new_contributor() );
-		$this->assertFalse( $attendee13->is_new_contributor() );
+		$this->assertTrue( $attendee11->is_new_contributor_legacy() );
+		$this->assertTrue( $attendee12->is_new_contributor_legacy() );
+		$this->assertFalse( $attendee13->is_new_contributor_legacy() );
```

Comment:
Is this just an intermediate step or are you planning to keep testing the legacy function? I'd argue in the above we should use the `Attendee_Adder` from #278 rather than directly inserting the attendee?

Reply:
Yes indeed, it's an intermediate step, I renamed it so that tests would continue passing while I work on the PR. Everything will be moved to the `Attendee_Adder` by the end of this PR, like you suggested.